code/game/mecha/mecha_control_console.dm

```diff
@@ -41,8 +41,14 @@
 		return
 	if(href_list["shock"])
 		var/obj/item/mecha_parts/mecha_tracking/MT = locate(href_list["shock"])
-		if (istype(MT))
+		if (istype(MT) && MT.chassis)
 			MT.shock()
+			//if(MT.chassis.occupant)

```

Comment:
What's up with the commented out code block?

Reply:
Forgot about that.